### DISCUSSION TITLE
Solved- Disabling controls of images

### DISCUSSION THREAD

**Discussion by turukas** [mozilla/hubs#4157] [General Questions & Discussion] [answered]
Hi,
Is it possible to disable (remove) hover interface of images added through spoke, so that they can't be interacted with in any way in Hubs? As an addition, can these images be removed from the Media list as well?
Thanks a lot in advance!

**Comment by turukas** [comment, marked as answer]
I apologise for posting the question before better exploring the already available options - image controls. So just please ignore the question, since I cannot remove it from the discussion.

**Reply by Softvision-GeluHaiduc** [reply]
Happy to hear that you could find the solution to your issue.
Marking this as answered to move it out of our pipeline, unmark the answer if you need any more information. Thank you!
